docs/docs/developer/pr-checklist.md

```diff
@@ -9,6 +9,11 @@ When contributing code through a pull request, please check the following:
 - [ ] `npm run check:svelte` (Type checking via SvelteKit)
 - [ ] `npm test` (unit tests)
 
+## Documentation
+
+- [ ] `npm run format` (formatting via Prettier)
```

Comment:
This does not actually fix the formatting errors, only `format:fix` does this. I'm not sure if that becomes clear here

Reply:
I see :D Maybe add the something about the existence of format:fix here as well then?